src/entities/calendar/calendar.resolver.ts

```diff
@@ -53,4 +60,28 @@ export class CalendarResolver {
   async deleteCalendar(@Args('_id', { type: () => ID }) _id: Types.ObjectId) {
     return this.service.delete(_id)
   }
+
+  //
+  // Field resolvers (for connections)
+  //
+
+  @ResolveField()
+  async calendarEventsConnection(
+    @Parent() calendar: CalendarDocument,
+    @Args() paginationArgs: PaginationArgs,
+  ) {
+    const filters = { calendar: calendar._id }
+    return this.calendarEventService.list(filters, paginationArgs)
+  }
+
+  @ResolveField()
+  async usersFollowerConnection(
+    @Parent() calendar: CalendarDocument,
+    @Args() paginationArgs: PaginationArgs,
+  ) {
+    const filters = { resourceId: calendar._id.toString() }
```

Comment:
👮 add resource name

```suggestion
    const filters = { resourceId: calendar._id.toString(), resourceName: Calendar.name }
```

Reply:
No, it's not necessary in this case. The only collection that needs that extra filter is Follower